easybuild/framework/easyblock.py

```diff
@@ -662,7 +662,8 @@ def collect_exts_file_info(self, fetch_files=True, verify_checksums=True):
                         if fetch_files:
                             src_path = self.obtain_file(src_fn, extension=True, urls=source_urls,
                                                         force_download=force_download,
-                                                        download_instructions=download_instructions)
+                                                        download_instructions=download_instructions,
+                                                        git_config=ext_options.get('git_config'))
```

Comment:
Hm, is this for the extensions to pick up git_config from `exts_default_options`? 

Because typically, that would be specific to each individual extension anyway, and a quick test
```
exts_list = [
    ('pytorch', '1.10.1', {
        'sources': [{'filename': SOURCE_TAR_GZ, 'git_config': {
            'url': 'https://github.com/pytorch',
            'repo_name': 'pytorch',
            'tag': 'v%(version)s',
            },
        }]
    }),
]
```
seems to download just fine for me without this change. I'm not actively opposed to it, but i can't imagine anyway this would ever apply to more than 1 extension anyway?

Reply:
No, I made the change to pick up `git_config` directly from the options map. I found no example such as yours, displaying that it was possible to put the key `git_config` under `sources`.

I have removed this commit from the Pull Request.